src/card.tsx

```diff
@@ -723,3 +729,151 @@ export const DisplayDataCard = React.forwardRef<HTMLDivElement, DisplayDataCardP
+    width?: number | string;
+    height?: number | string;
+    icon?: React.ReactElement;
+    actions?: Array<CardAction>;
+    onClose?: () => void;
+    dataAttributes?: DataAttributes;
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    title?: string;
+    titleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+}
+
+const POSTER_CARD_MIN_WIDTH = 140;
+const POSTER_CARD_MIN_HEIGHT = 112;
+export const PosterCard = React.forwardRef<HTMLDivElement, PosterCardProps>(
+    (
+        {
+            dataAttributes,
+            backgroundImage,
+            width,
+            height,
+            aspectRatio = '7:10',
+            ariaLabel,
+            actions,
+            onClose,
+            icon,
+            headline,
+            pretitle,
+            pretitleLinesMax,
+            title,
+            titleLinesMax,
+            description,
+            descriptionLinesMax,
+        },
+        ref
+    ) => {
+        const withGradient = !!backgroundImage;
+        const textShadow = withGradient ? '0 0 16px rgba(0,0,0,0.4)' : undefined;
+        const hasTopActions = actions?.length || onClose;
+        const {textPresets} = useTheme();
+
+        return (
+            <MaybeWithActions
+                width={width}
+                height={height}
+                minWidth={POSTER_CARD_MIN_WIDTH}
+                minHeight={POSTER_CARD_MIN_HEIGHT}
+                aspectRatio={aspectRatio}
+                onClose={onClose}
+                actions={actions}
+                aria-label={ariaLabel}
+                isInverse
+            >
+                <InternalBoxed
+                    borderRadius={16}
+                    className={styles.boxed}
+                    dataAttributes={dataAttributes}
+                    ref={ref}
+                    width="100%"
+                    minHeight="100%"
+                    isInverse
+                    background={backgroundImage ? vars.colors.backgroundContainer : undefined}
+                >
+                    <div
+                        className={styles.displayCard}
+                        style={{
+                            backgroundImage: backgroundImage
+                                ? `url("${CSS.escape(backgroundImage)}")`
+                                : undefined,
+                            paddingTop: withGradient && !icon && !hasTopActions ? 0 : 24,
+                        }}
+                    >
+                        {icon ? (
+                            <Box paddingBottom={withGradient ? 0 : 40} paddingX={24}>
+                                {icon}
+                            </Box>
+                        ) : (
+                            <Box paddingBottom={actions?.length || onClose ? (withGradient ? 24 : 64) : 0} />
+                        )}
+                        <Box
+                            paddingX={24}
```

Comment:
```suggestion
                            paddingX={16}
```

Reply:
hmm.. u sure u want this? it will affect other components (see the failing screenshots below)